addons/mail/models/mail_gateway_allowed.py

```diff
@@ -35,7 +36,7 @@ def get_empty_list_help(self, help_message):
         LOOP_MINUTES = int(get_param('mail.gateway.loop.minutes', 120))
         LOOP_THRESHOLD = int(get_param('mail.gateway.loop.threshold', 20))
 
-        return _('''
+        return Markup(_('''
```

Comment:
Markup is useless here (we still pass the argument to `_` not to `Markup`, and they are `int` so it's ok)

or if we really want to return a Markup object, we should give the argument to Markup itself (not `_`)

Reply:
Done, just to encore usage of markup I kept the Markup and gave parameters to him.